lib/config/logger.go

```diff
@@ -19,18 +19,18 @@ package config
 import (
 	"errors"
 	"fmt"
-	"path"
-
 	"github.com/influxdata/influxdb/toml"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 	"gopkg.in/natefinch/lumberjack.v2"
+	"path"
+	"path/filepath"
```

Comment:
Please adjust the order of referenced packages.

OpenGemini use https://github.com/incu6us/goimports-reviser tool to automatic sorting